### DISCUSSION TITLE
'Column not found' when ordering with BelongsTo relationship

### DISCUSSION THREAD

**Discussion by Restingo** [rappasoft/laravel-livewire-tables#318] [Q&A] [answered]
I've got a person that has a salutation (BelongsTo). I've added the column like this:

`Column::make('Salutation', 'salutation.name')->sortable()->searchable(),`

but when I try to order it I get the following error:

`Illuminate\Database\QueryException
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'salutation.name' in 'order clause'`

How can I sort with relationship columns?

**Comment by rappasoft** [comment, marked as answer]
https://github.com/rappasoft/laravel-livewire-tables/wiki/Using-the-built-in-sorting

See the last example.

**Reply by Restingo** [reply]
Thank you, I'll try!
